SignInCheckIn/SignInCheckIn/Controllers/ChildSigninController.cs

```diff
@@ -210,7 +207,8 @@ public IHttpActionResult PrintParticipant(int eventParticipantId)
 
                 try
                 {
-                    return Ok(_childSigninService.PrintParticipant(eventParticipantId, kioskIdentifier, token));
+                    if (!authDto.Authorization.MpRoles.ContainsKey(_kidsClubTools)) throw new UnauthorizedAccessException();
```

Comment:
Logic executed after an if statement should be in a code block, unless it's a ternary operator - 

if (condition)
{
 // do something
}